### DISCUSSION TITLE
Reload vcd file

### DISCUSSION THREAD

**Discussion by hi0t** [Lramseyer/vaporview#11] [Ideas]
I couldn't find a button or command to reload the vcd. If there is no such function, it would be a good idea to add it. If the data file has been updated, you have to close it, then open it and configure viewing again.

**Comment by Lramseyer** [comment]
Great idea! I just added it to the issues list as a feature request and tagged you in it. Feel free to discuss further on the issue!

[https://github.com/Lramseyer/vaporview/issues/12](https://github.com/Lramseyer/vaporview/issues/12)
